Lib/_pyrepl/_module_completer.py

```diff
@@ -96,8 +96,14 @@ def _find_modules(self, path: str, prefix: str) -> list[str]:
             modules = [mod_info for mod_info in modules
                        if mod_info.ispkg and mod_info.name == segment]
             modules = self.iter_submodules(modules)
+
         return [module.name for module in modules
-                if module.name.startswith(prefix)]
+                if self._should_add_module_name(module.name, prefix)]
+
+    def _should_add_module_name(self, module_name: str, prefix: str) -> bool:
```

Comment:
```suggestion
    def _is_suggestion_match(self, module_name: str, prefix: str) -> bool:
```
I find the current one is quite ambiguous, this should clear it up.

Reply:
I agree

Reply:
@kevteg You do not have to use suggestions, especially like mine here where there is other work attached to it, it is often simpler to just send one commit yourself:-)